core/build.gradle.kts

```diff
@@ -81,6 +83,8 @@ dependencies {
     testImplementation(kotlin("test"))
     testImplementation(kotlin("test-junit5"))
     testRuntimeOnly("org.slf4j:slf4j-log4j12")
+    testAnnotationProcessor("org.immutables:value:2.8.8")
```

Comment:
Probably should have same immutables version for `annotationProcessor` and `testAnnotationProcessor`.

Reply:
Ultimately versions go in root directory's `build.properties`. You will need to declare the dependency in `bom/build.gradle.kts`.